styles/itsy.css

```diff
@@ -0,0 +1,93 @@
+body {
+  margin: 0px;
+}
+.home {
+  display: flex;
+  background: url(../img/lead.jpg);
+  height: 100vh;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  padding: 10px 30px;
+  font-size: 30px;
+  background: rgba(255,255,255, 0.8);
+}
+
+.amatic {
+    font-family: 'Amatic SC', 'Heveltica Neue', Arial, sans-serif;
+}
+
+.header__list {
+  list-style: none;
+  float: left;
+  text-transform: uppercase;
+  margin-left: 30px;
+  font-size: 25px;
+}
+
+.header__list:hover {
+  color: #e68a8a;
```

Comment:
please, identify classes via comma to reduce duplication of styles.
`.some-class, .b-class {color: red}`